.github/workflows/README.md

```diff
@@ -0,0 +1,28 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+| Workflow name | Trigger Phrase | Cron Status |
+|:-------------:|:--------------:|:-----------:|
+| [ Go Precommit ](https://github.com/apache/beam/actions/workflows/beam_PreCommit_Go.yml) | `Run Go PreCommit`| [![Go Precommit](https://github.com/apache/beam/actions/workflows/beam_PreCommit_Go.yml/badge.svg?event=schedule)](https://github.com/apache/beam/actions/workflows/beam_PreCommit_Go.yml) |
+| [ Python PreCommit Docker ](https://github.com/apache/beam/actions/workflows/job_Precommit_Python_DockerBuild.yml) | `Run PythonDocker PreCommit`| [![.github/workflows/job_Precommit_Python_DockerBuild.yml](https://github.com/apache/beam/actions/workflows/job_Precommit_Python_DockerBuild.yml/badge.svg?event=schedule)](https://github.com/apache/beam/actions/workflows/job_Precommit_Python_DockerBuild.yml) |
+| [ Python Precommit Formatter ](https://github.com/apache/beam/actions/workflows/job_PreCommit_PythonAutoformatter.yml) | `Run PythonFormatter PreCommit`| [![.github/workflows/job_Precommit_Python_DockerBuild.yml](https://github.com/apache/beam/actions/workflows/job_Precommit_Python_DockerBuild.yml/badge.svg?event=schedule)](https://github.com/apache/beam/actions/workflows/job_Precommit_Python_DockerBuild.yml) |
```

Comment:
The job is `Python Precommit Formatter`, but the links are related to the `Python PreCommit Docker`

Reply:
Fixed